modules/core/shared/src/main/scala/lucuma/core/model/sequence/StepConfig.scala

```diff
@@ -119,45 +110,43 @@ object StepConfig {
     /** @group Optics */
     val shutter: Lens[Gcal, GcalShutter] =
       Focus[Gcal](_.shutter)
-  }
-
-  final case class Science(offset: Offset, guiding: StepGuideState) extends StepConfig(StepType.Science)
-
-  object Science {
-    given Eq[Science] =
-      Eq.by { a => (
-        a.offset,
-        a.guiding
-      )}
 
     /** @group Optics */
-    val offset: Lens[Science, Offset] =
-      Focus[Science](_.offset)
+    val telescope: Lens[Gcal, TelescopeConfig] =
+      Focus[Gcal](_.telescope)
+  end Gcal
 
-    /** @group Optics */
-    val guiding: Lens[Science, StepGuideState] =
-      Focus[Science](_.guiding)
-  }
+  final case class Science(
+    telescope: TelescopeConfig
```

Comment:
Switch to embed `TelescopeConfig` here.  I anticipate changing the GraphQL API accordingly.